### PR TITLE
Add support for Django 1.11, 2.0, 2.1, and 2.2, and drop support for older versions

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

**Comment by DmytroLitvinov** [comment]
Hello @vxsx @czpython @FinalAngel @ojii .

Could you please merge it and release at PyPi?

**Comment by DmytroLitvinov** [comment]
Related issues/PR:
https://github.com/divio/django-sekizai/issues/71
https://github.com/divio/django-sekizai/pull/75

**Comment by DmytroLitvinov** [comment]
Everything seems ready to be merged.

**Comment by JeremyGibson** [comment]
Noticed that this build passes https://travis-ci.org/divio/django-sekizai/builds/502073740. Should the CircleCI be disconnected?

**Comment by DmytroLitvinov** [comment]
@JeremyGibson we should be more worried about a release, which is still not released for Django 1.11(LTS) version which was released almost 2 years ago(4th of April 2017)

Any news @vxsx @czpython @FinalAngel @ojii?
Still a lot of projects depend on this package.

**Comment by czpython** [comment]
ping @vThaian

**Comment by DmytroLitvinov** [comment]
Even django-cms, which is under your development, depends on this library.

File: .travis.yml [hunk 1]
```diff
@@ -1,45 +1,30 @@
+dist: xenial
 language: python
 sudo: false
 python:
  - 2.7
- - 3.3
  - 3.4
  - 3.5
+ - 3.6
```
Comment (vThaian) [inline .travis.yml]: Would be great to add python 3.7 for 2.0+

**Comment by DmytroLitvinov** [comment]
Ping @czpython for merging into master and publish it to PyPi

**Comment by vThaian** [comment]
Thanks @timgraham for changes. @FinalAngel could you please make a release?

**Comment by DmytroLitvinov** [comment]
Hello @vThaian and @FinalAngel .
Do you need some help with release?

**Comment by FinalAngel** [comment]
@DmytroLitvinov we just have to get to it. I hope to be able to release this week :)

**Comment by DmytroLitvinov** [comment]
Nice :) we will wait for official PyPi release.

**Comment by FinalAngel** [comment]
@DmytroLitvinov is [has been released](https://pypi.org/project/django-sekizai/1.0.0/#history) :)

**Pull request merged**